src/Coverage/CoverageCodeLensProvider.ts

```diff
@@ -2,15 +2,18 @@ import * as vscode from 'vscode'
 
 import { JestExt } from '../JestExt'
 
+type GetJestExt = (uri: vscode.Uri) => JestExt | undefined
+
 export class CoverageCodeLensProvider implements vscode.CodeLensProvider {
-  private getJestExt: (uri: vscode.Uri) => JestExt
+  private getJestExt: GetJestExt
 
-  constructor(getJestExt: (uri: vscode.Uri) => JestExt) {
+  constructor(getJestExt: GetJestExt) {
     this.getJestExt = getJestExt
   }
 
   public provideCodeLenses(document: vscode.TextDocument, _token: vscode.CancellationToken) {
-    const coverage = this.getJestExt(document.uri).coverageMapProvider.getFileCoverage(document.fileName)
+    const ext = this.getJestExt(document.uri)
+    const coverage = ext ? ext.coverageMapProvider.getFileCoverage(document.fileName) : undefined

```

Comment:
Since we return if `coverage` is falsy, what about `const coverage = ext && ext.coverageMapProvider.getFileCoverage(document.fileName)` instead?

Reply:
great idea!